tests/x11regressions/evolution/evolution_mail_ews.pm

```diff
@@ -0,0 +1,128 @@
+#Test Case #1503965: Evolution: Setup MS Exchange account
+use strict;
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $mailbox     = 'zzzSUSEExTest19@microfocus.com';
+    my $mail_passwd = 'P@$$w0rd2015';
+
+    mouse_hide(1);
+
+    # Clean and Start Evolution
+    x11_start_program("xterm -e \"killall -9 evolution; find ~ -name evolution | xargs rm -rf;\"");
+    x11_start_program("evolution");
+    if (check_screen "evolution-default-client-ask") {
+        assert_and_click "evolution-default-client-agree";
+    }
+
+    # Follow the wizard to setup mail account
+    assert_screen "test-evolution-1";
```

Comment:
Hi okurz,

Thanks for the recommendation.  I tried to replace the api check_screen with match_has_tag, but the test script didn't work as expected. After digging into the definition of match_has_tag function, I find it is improper to be used here. Because the precondition of using match_has_tag is that there must be a matched needle previously, and then we can judge if there is a wanted tag in the matched needle. But here, I want to match a tag from a new screen (which is not 100% sure to pop up), so match_has_tag doesn't work in this case.